source/uwp/Renderer/idl/AdaptiveCards.Uwp.idl

```diff
@@ -1945,6 +1976,28 @@ namespace AdaptiveCards
             [default] interface IAdaptiveRenderArgs;
         };
 
+        [
+            uuid(c0222295-bbec-4ff6-bdf3-f207ce78df69),
+            version(NTDDI_WIN10_RS1)
+        ]
+        interface IAdaptiveFailure : IInspectable
+        {
+            [propget] HRESULT FailureStatusCode([out, retval] FailureStatusCode* value);
+            [propput] HRESULT FailureStatusCode([in] FailureStatusCode value);
```

Comment:
>propput [](start = 13, length = 7)

I think the putters are not needed. I'll see if I need them for once I finish integrating the shared model warnings/errors.